internal/plugin/plugin.go

```diff
@@ -28,10 +31,13 @@ type Plugin struct {
 	bufpluginconfig.ExternalConfig `yaml:"-"`
 	// Plugin identity (parsed from ExternalConfig.Name).
 	Identity bufpluginref.PluginIdentity `yaml:"-"`
+	// For callers that need git commit info - ensure we only calculate it once.
+	gitCommitOnce sync.Once `yaml:"-"`
+	gitCommit     string    `yaml:"-"`
 }
 
 func (p *Plugin) String() string {
-	return fmt.Sprintf("%+v", *p)
+	return fmt.Sprintf("%s:%s", p.Identity.IdentityString(), p.Version)
```

Comment:
I'm assuming nothing was relying on the old `String` output? (Just for debugging?)

Reply:
Yep was just for debugging. It broke when I added a sync.Once here.